common/scripts/install-run-rush.js

```diff
@@ -1,3 +1,7 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
You're going to want to merge with `master` then do the following to fix this;

1. Comment out line 9 in `common/git-hooks/pre-commit`
1. Run `rush update`
1. Commit the changes to the two files in this folder
1. Uncomment the line from #1
1. Run `rush install` again

Reply:
What line from that PR are you referring to? All I see in that PR is spacing in a doc.